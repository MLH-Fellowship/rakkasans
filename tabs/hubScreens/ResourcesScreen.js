import React from "react";
import { View, Text, StyleSheet, Image, TouchableOpacity } from "react-native";
import { useNavigation } from "@react-navigation/native";

import SquareButton from "../../components/SquareButton";
import Colors from "../../constants/Colors";

export default function ProgramsScreen() {
  const navigation = useNavigation();
  return (
    <View style={styles.container}>
      <Image
        source={require("../../assets/images/Torri.png")}
        style={{ width: 30, height: 30, top: "-8%" }}
      />
      <View style={[styles.buttonView, { paddingTop: 10 }]}>
        <SquareButton
          name="ribbon"
          text="SHARP"
          buttonSize={65}
          textSize={12}
          iconSize={38}
          onPress={() =>
            navigation.navigate("Resource", {
              name: "SHARP",
              // image: require("../../assets/images/HHC.png"),
            })
          }
        />
        <SquareButton
          name="bank"
          text="EO"
          buttonSize={65}
          textSize={12}
          iconSize={38}
          onPress={() =>
            navigation.navigate("Resource", {
              name: "EO",
              // image: require("../../assets/images/HHC.png"),
            })
          }
        />
      </View>
      <View style={styles.buttonView}>
        <SquareButton
          name="home-group"
          text="FRG"
          buttonSize={65}
          textSize={12}
          iconSize={38}
          onPress={() =>
            navigation.navigate("Resource", {
              name: "FRG",
              // image: require("../../assets/images/HHC.png"),
            })
          }
        />
        <SquareButton
          name="basketball"
          text="MWR"
          buttonSize={65}
          textSize={12}
          iconSize={38}
          onPress={() =>
            navigation.navigate("Resource", {
              name: "MWR",
              // image: require("../../assets/images/HHC.png"),
            })
          }
        />
      </View>
      <View style={styles.buttonView}>
        <SquareButton
          name="human-handsup"
          text="BOSS"
          buttonSize={65}
          textSize={12}
          iconSize={38}
          onPress={() =>
            navigation.navigate("Resource", {
              name: "BOSS",
              // image: require("../../assets/images/HHC.png"),
            })
          }
        />
        <SquareButton
          name="school"
          text="Education"
          buttonSize={65}
          textSize={11}
          iconSize={38}
          onPress={() =>
            navigation.navigate("Resource", {
              name: "Education",
              // image: require("../../assets/images/HHC.png"),
            })
          }
        />
      </View>
      <View style={[styles.buttonView, { paddingBottom: 25 }]}>
        <SquareButton
          name="flower"
          text="ACS"
          buttonSize={65}
          textSize={12}
          iconSize={38}
          onPress={() =>
            navigation.navigate("Resource", {
              name: "ACS",
              // image: require("../../assets/images/HHC.png"),
            })
          }
        />
        <SquareButton
          name="account-group"
          text="PMO"
          buttonSize={65}
          textSize={13}
          iconSize={38}
          onPress={() =>
            navigation.navigate("Resource", {
              name: "PMO",
              // image: require("../../assets/images/HHC.png"),
            })
          }
        />
      </View>
      <View style={[styles.buttonView, { paddingBottom: 25 }]}>
        <SquareButton
          name="calendar-clock"
          text="Retention"
          buttonSize={65}
          textSize={11}
          iconSize={38}
          onPress={() =>
            navigation.navigate("Resource", {
              name: "Retention",
              // image: require("../../assets/images/HHC.png"),
            })
          }
        />
        <SquareButton
          name="christianity-outline"
          text="Religious"
          buttonSize={65}
          textSize={11}
          iconSize={38}
          onPress={() =>
            navigation.navigate("Resource", {
              name: "Religious Services",
              // image: require("../../assets/images/HHC.png"),
            })
          }
        />
      </View>
      <View style={[styles.buttonView, { paddingBottom: 25 }]}>
        <SquareButton
          name="select-group"
          text="ADAM BAE"
          buttonSize={65}
          textSize={11}
          iconSize={38}
          onPress={() =>
            navigation.navigate("Resource", {
              name: "ADAM BAE",
              // image: require("../../assets/images/HHC.png"),
            })
          }
        />
      </View>
    </View>
  );
}
const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: Colors.white,
    width: "100%",
    alignItems: "center",
    justifyContent: "center",
  },
  buttonView: {
    width: "90%",
    flexDirection: "row",
    alignItems: "center",
    justifyContent: "space-evenly",
    paddingBottom: 20,
  },
});
