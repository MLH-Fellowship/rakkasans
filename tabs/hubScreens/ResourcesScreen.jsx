import React, { useEffect, useState } from 'react';
import { View, StyleSheet, Image } from 'react-native';
import { useNavigation } from '@react-navigation/native';
import axios from 'axios';
import SquareButton from '../../components/SquareButton';
import Colors from '../../constants/Colors';

export default function ResourcesScreen() {
  const navigation = useNavigation();
  const [resources, setResources] = useState([]);

  useEffect(() => {
    const getArmyResources = async () => {
      try {
        const request = await axios.get('http://localhost:3001/resources');
        console.log(request.data);
        const armyResources = request.data.filter(
          (data) => data.resource_type === 'army',
        );
        setResources(armyResources);
      } catch (error) {
        console.error(error);
      }
    };

    getArmyResources();
  }, []);

  const styles = StyleSheet.create({
    container: {
      flexWrap: 'wrap',
      alignItems: 'center',
      flexDirection: 'row',
      justifyContent: 'center',
      backgroundColor: Colors.white,
      width: '100%',
    },
    buttonView: {
      width: '45%',
      alignItems: 'center',
      justifyContent: 'space-evenly',
      paddingBottom: 30,
    },
    image: {
      marginTop: '5%',
      width: '100%',
    },
  });
  return (
    <View style={styles.container}>
      {console.log(resources)}
      <View style={styles.image}>
        <Image
          source={require('../../assets/images/Torri.png')}
          style={{
            width: 50,
            height: 50,
            paddingTop: '10%',
            alignSelf: 'center',
          }}
        />
      </View>
      {resources.map(
        ({
          id,
          image,
          icon_name,
          resource_title,
          phone_number,
          email,
          grad_times,
          location,
          report_times,
        }) => (
          <View style={[styles.buttonView]}>
            <SquareButton
              id={id}
              name={icon_name}
              text={resource_title}
              buttonSize={65}
              textSize={13}
              iconSize={38}
              onPress={() => navigation.navigate('Resource', {
                name: resource_title,
                image,
                email,
                location,
                phone: phone_number,
                grad_times,
                report_times,
                // image: require("../../assets/images/HHC.png"),
              })}
            />
          </View>
        ),
      )}
    </View>
  );
}
