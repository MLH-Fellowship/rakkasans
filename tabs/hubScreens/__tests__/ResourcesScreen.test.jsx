import React from 'react';
import renderer from 'react-test-renderer';

import ResourcesScreen from '../ResourcesScreen';

jest.useFakeTimers();
jest.mock('@react-navigation/native');

it('renders correctly', () => {
  const tree = renderer.create(<ResourcesScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
