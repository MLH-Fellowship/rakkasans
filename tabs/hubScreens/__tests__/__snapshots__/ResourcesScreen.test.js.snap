// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ffffff",
      "flex": 1,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 30,
        "top": "-8%",
        "width": 30,
      }
    }
  />
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-evenly",
          "paddingBottom": 20,
          "width": "90%",
        },
        Object {
          "paddingTop": 10,
        },
      ]
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 15,
          "elevation": 5,
          "opacity": 1,
          "padding": 10,
          "shadowColor": "#a0a0a0",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.4,
          "shadowRadius": 8,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 65,
              "width": 65,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#333333",
                "paddingBottom": 6,
                "textAlign": "center",
              },
              Object {
                "fontFamily": "andale-mono",
              },
              Object {
                "fontSize": 12,
              },
            ]
          }
        >
          SHARP
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 19,
              "height": 38,
              "justifyContent": "center",
              "width": 38,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 15,
          "elevation": 5,
          "opacity": 1,
          "padding": 10,
          "shadowColor": "#a0a0a0",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.4,
          "shadowRadius": 8,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 65,
              "width": 65,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#333333",
                "paddingBottom": 6,
                "textAlign": "center",
              },
              Object {
                "fontFamily": "andale-mono",
              },
              Object {
                "fontSize": 12,
              },
            ]
          }
        >
          EO
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 19,
              "height": 38,
              "justifyContent": "center",
              "width": 38,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-evenly",
        "paddingBottom": 20,
        "width": "90%",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 15,
          "elevation": 5,
          "opacity": 1,
          "padding": 10,
          "shadowColor": "#a0a0a0",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.4,
          "shadowRadius": 8,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 65,
              "width": 65,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#333333",
                "paddingBottom": 6,
                "textAlign": "center",
              },
              Object {
                "fontFamily": "andale-mono",
              },
              Object {
                "fontSize": 12,
              },
            ]
          }
        >
          FRG
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 19,
              "height": 38,
              "justifyContent": "center",
              "width": 38,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 15,
          "elevation": 5,
          "opacity": 1,
          "padding": 10,
          "shadowColor": "#a0a0a0",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.4,
          "shadowRadius": 8,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 65,
              "width": 65,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#333333",
                "paddingBottom": 6,
                "textAlign": "center",
              },
              Object {
                "fontFamily": "andale-mono",
              },
              Object {
                "fontSize": 12,
              },
            ]
          }
        >
          MWR
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 19,
              "height": 38,
              "justifyContent": "center",
              "width": 38,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-evenly",
        "paddingBottom": 20,
        "width": "90%",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 15,
          "elevation": 5,
          "opacity": 1,
          "padding": 10,
          "shadowColor": "#a0a0a0",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.4,
          "shadowRadius": 8,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 65,
              "width": 65,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#333333",
                "paddingBottom": 6,
                "textAlign": "center",
              },
              Object {
                "fontFamily": "andale-mono",
              },
              Object {
                "fontSize": 12,
              },
            ]
          }
        >
          BOSS
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 19,
              "height": 38,
              "justifyContent": "center",
              "width": 38,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 15,
          "elevation": 5,
          "opacity": 1,
          "padding": 10,
          "shadowColor": "#a0a0a0",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.4,
          "shadowRadius": 8,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 65,
              "width": 65,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#333333",
                "paddingBottom": 6,
                "textAlign": "center",
              },
              Object {
                "fontFamily": "andale-mono",
              },
              Object {
                "fontSize": 11,
              },
            ]
          }
        >
          Education
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 19,
              "height": 38,
              "justifyContent": "center",
              "width": 38,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-evenly",
          "paddingBottom": 20,
          "width": "90%",
        },
        Object {
          "paddingBottom": 25,
        },
      ]
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 15,
          "elevation": 5,
          "opacity": 1,
          "padding": 10,
          "shadowColor": "#a0a0a0",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.4,
          "shadowRadius": 8,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 65,
              "width": 65,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#333333",
                "paddingBottom": 6,
                "textAlign": "center",
              },
              Object {
                "fontFamily": "andale-mono",
              },
              Object {
                "fontSize": 12,
              },
            ]
          }
        >
          ACS
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 19,
              "height": 38,
              "justifyContent": "center",
              "width": 38,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 15,
          "elevation": 5,
          "opacity": 1,
          "padding": 10,
          "shadowColor": "#a0a0a0",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.4,
          "shadowRadius": 8,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 65,
              "width": 65,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#333333",
                "paddingBottom": 6,
                "textAlign": "center",
              },
              Object {
                "fontFamily": "andale-mono",
              },
              Object {
                "fontSize": 13,
              },
            ]
          }
        >
          PMO
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 19,
              "height": 38,
              "justifyContent": "center",
              "width": 38,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-evenly",
          "paddingBottom": 20,
          "width": "90%",
        },
        Object {
          "paddingBottom": 25,
        },
      ]
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 15,
          "elevation": 5,
          "opacity": 1,
          "padding": 10,
          "shadowColor": "#a0a0a0",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.4,
          "shadowRadius": 8,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 65,
              "width": 65,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#333333",
                "paddingBottom": 6,
                "textAlign": "center",
              },
              Object {
                "fontFamily": "andale-mono",
              },
              Object {
                "fontSize": 11,
              },
            ]
          }
        >
          Retention
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 19,
              "height": 38,
              "justifyContent": "center",
              "width": 38,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 15,
          "elevation": 5,
          "opacity": 1,
          "padding": 10,
          "shadowColor": "#a0a0a0",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.4,
          "shadowRadius": 8,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 65,
              "width": 65,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#333333",
                "paddingBottom": 6,
                "textAlign": "center",
              },
              Object {
                "fontFamily": "andale-mono",
              },
              Object {
                "fontSize": 11,
              },
            ]
          }
        >
          Religious
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 19,
              "height": 38,
              "justifyContent": "center",
              "width": 38,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-evenly",
          "paddingBottom": 20,
          "width": "90%",
        },
        Object {
          "paddingBottom": 25,
        },
      ]
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 15,
          "elevation": 5,
          "opacity": 1,
          "padding": 10,
          "shadowColor": "#a0a0a0",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.4,
          "shadowRadius": 8,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 65,
              "width": 65,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#333333",
                "paddingBottom": 6,
                "textAlign": "center",
              },
              Object {
                "fontFamily": "andale-mono",
              },
              Object {
                "fontSize": 11,
              },
            ]
          }
        >
          ADAM BAE
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderRadius": 19,
              "height": 38,
              "justifyContent": "center",
              "width": 38,
            }
          }
        >
          <Text />
        </View>
      </View>
    </View>
  </View>
</View>
`;
